iwyu.cc

```diff
@@ -1228,7 +1226,7 @@ class IwyuBaseAstVisitor : public BaseAstVisitor<Derived> {
       : Base(visitor_state->compiler),
         visitor_state_(visitor_state) {}
 
-  virtual ~IwyuBaseAstVisitor() {}
+  ~IwyuBaseAstVisitor() override {}
```

Comment:
`IwyuBaseAstVisitor` is intended to have subclasses. Isn't it better to have virtual destructor in this case? Even if it's not used at the moment? I didn't check if it's actually used.